### ISSUE TITLE
A11y - Help page

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1378]
## User story

- contrast issues: breadcrumbs, links, tab labels
- Headings issues: missing h1 and skipping heading levels
- A11y tabs: make sure tabs are a11y: https://www.w3.org/WAI/ARIA/apg/patterns/tabs/
- The tab content could be divided into `<section>`s
- Giga DB search tab:
  - Undue usage of table: There is an item and a definitiion, and no headers, dl pattern might be better here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl
![image](https://github.com/gigascience/gigadb-website/assets/143437854/eed6ad2d-68c7-41d8-9ac9-4e6a59b4ba91)
  - Circular label lacks a11y name: The circular labels should have an a11y name to indicate what they represent. These are displayed in other pages, so they should be a common component
- Submission guidelines tab
  - Link to excel (this an the other tabs): Make sure all links to excels are indicated that this is an excel file
- Controlled vocabulary tab
  - use <dl> for lists of definitions
![image](https://github.com/gigascience/gigadb-website/assets/143437854/2775a0db-84b3-4411-bcad-30df4447143c)
![image](https://github.com/gigascience/gigadb-website/assets/143437854/942a32e7-3a16-47a7-b6dc-9bc73d3eb47c)
- API tab
  - Make examples a `<ol>`
![image](https://github.com/gigascience/gigadb-website/assets/143437854/5c7f94b5-3b93-4ce1-8fd8-b79acb7b5d94)
  - Use semantic `<code>` for cli commands
![image](https://github.com/gigascience/gigadb-website/assets/143437854/9e6aed2d-141b-460f-93c3-543bcac25617)

https://beta.gigadb.org/site/help

## Acceptance criteria

- [] ~~Solve contrast issues (prio 3) (there's some work to be completed and merged in this branch, before handling this https://github.com/gigascience/gigadb-website/issues/1374)~~ (fixed in PR #1427)
- [x] Fix heading hierarchy issues (prio 3)
- [ ] A11y tabs (prio 1)
- [x] Use sections to divide the tab content (prio 1)
- [x] Use appropriate html for definition lists `<dl>` (prio 2)
- [x] Symbols representing gigadb categories (Circular label with single letter) have a11y labels (prio 3)
- [x] Indicate to assistive tech that link to excel files are links to excel (prio 1)
- [x] Wrap examples list in `<ol>` (prio 2)
- [x] Use semantic `<code>` element for cli commands (prio 1)
